### PR TITLE
Update helm chart configs and readme

### PR DESCRIPTION
**Pull request by fuziontech**
Update helm chart configs and readme. Tested on local kubernetes cluster with nfs pv class.

### CODE REVIEW

File: dev/helm/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+              containerPort: 3000
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /
+              port: http
```
Comment (NGPixel) [inline dev/helm/templates/deployment.yaml:63]: The `/healthz` endpoint would be better suited.

```suggestion
          livenessProbe:
            httpGet:
              path: /healthz
              port: http
          readinessProbe:
            httpGet:
              path: /healthz
              port: http
```

**Comment by don-sh** [comment]
Shouldn't there be an option to set the DB_TYPE if postgres is disabled?

**Comment by NGPixel** [comment]
> Shouldn't there be an option to set the DB_TYPE if postgres is disabled?

No, as PostgreSQL is mandatory for high-availability.

**Comment by don-sh** [comment]
> > Shouldn't there be an option to set the DB_TYPE if postgres is disabled?
> 
> No, as PostgreSQL is mandatory for high-availability.

Then you need to remove the {{ if .Values.postgresql.enabled }} below - otherwise you run into an "invalid DB_TYPE".
```
            {{ if .Values.postgresql.enabled }}
            - name: DB_TYPE
              value: postgres
            {{ end }}
```

**Comment by igorfernandes** [comment]
Any update about this PR?

**Comment by fuziontech** [comment]
need to update with feedback. I'll do that tonight

**Comment by fuziontech** [comment]
@don-sh - You are right, updated `deployment.yaml` to only support postgres and not remove that config if `postgres.enabled` is set false

**Pull request merged**